docs/en/stack/ml/df-analytics/dfa-regression.asciidoc

```diff
@@ -0,0 +1,103 @@
+[role="xpack"]
+[[dfa-regression]]
+== {regression-cap}
+
+{reganalysis-cap} is a {ml} process for estimating the relationships among 
+a number of {feature-vars} and a {depvar} (or {con-targvar}), then making 
+further predictions based on the described relationship.
+
+For example, suppose we are interested in finding the relationship between 
+apartment size and monthly rent in a city. In this simple example, the rent is 
+the {depvar} while the apartment size is the {feature-var} and we try to 
+find out the relationship between them in order to make further predictions. Our 
+imaginary dataset consists of three data points:
+
+|===
+| Size (m2) | Monthly rent 
+| 44        | 1600
+| 24        | 1055
+| 63        | 2300
+|===
+
+After the model described the relationship between the {feature-vars} and the 
+{depvar}, it can predict the monthly rent of a hundred square meter-size 
+apartment.
+
+Of course, this is a very simple example, usually {regression} problems are 
+multi-dimensional, so the relationships that {reganalysis} tries to find are 
+between multiple {feature-vars} and the {depvar}. To extend our example, a 
+more complex {reganalysis} could take into account further {feature-vars} like 
+the location of the apartment in the city, on which floor it is, and whether the 
+apartment has a riverside view or not and so on.
+
+
+[discrete]
+[dfa-regression-features]
+=== {feature-vars-cap}
+
+{feature-vars-cap} are the values that the {con-targvar} value depends on 
```

Comment:
I see we're using `dependent variable` and `continuous target variable` interchangeably. I think users not familiar with the terminology might get confused. I suggest we stick to `dependent variable` as that's what our API uses. In the description of `dependent variable` we can mention it is also known as `continuous target variable` for the more knowledgeable users.